src/main/java/com/google/sps/data/HiddenGem.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** Hidden Gems are restaurants or cafes in Australia which have 10-50 reviews and a rating of at least 3.5/5. */
+public final class HiddenGem {
+
+  private final int id;
```

Comment:
This is a matter of preference so feel free to ignore, but I'd make these fields public and remove the getters. Since they are final they can't be modified anyways.

It's a pattern I'd use for data classes in general (that don't have any behavior or internal state that you want to avoid exposing to the user), though you have to be careful with mutable objects like lists (since even though the var can't be modified, the object it contains can be).